linux/LinuxProcessList.c

```diff
@@ -56,6 +56,11 @@ in the source distribution for its full text.
 #include "LibSensors.h"
 #endif
 
+// CentOS 6's kernel doesn't provide a definition of O_PATH
+#ifndef O_PATH
+# define O_PATH 010000000
```

Comment:
Is this octal number intentional? Or should it read `0x10000000` instead?
```suggestion
# define O_PATH 0x10000000
```

Reply:
Yes, this matches the way it's defined in Linux:

https://github.com/torvalds/linux/blob/6f0d349d922ba44e4348a17a78ea51b7135965b1/include/uapi/asm-generic/fcntl.h#L84-L86

Reply:
Can you add this reference somehow for documentation?

Like: `// based on definition taken from uapi/asm-generic/fcnth.h in Linux kernel tree`

Just so that people have a reference where this comes from.